### ISSUE TITLE
[Bug]  One-way audio transmission with WebRTC adapter

### ISSUE THREAD

**Issue by djoksimo** [networked-aframe/networked-aframe#194]
Hi 👋 ,

I tried out https://haydenlee.io/networked-aframe/ and I also ran the [example](https://github.com/networked-aframe/networked-aframe/blob/master/examples/basic-audio.html
) from source:
`git clone, npm install, npm run dev` (served over HTTPS)

I tested this [example that uses the EasyRTC adapter instead of the WebRTC adapter](https://glitch.com/edit/#!/easyrtc-version) and it seems to work (but not on iOS or Safari) and it's deprecated (as mentioned in your repo).

## Steps
1. Set up basic socket.io signalling as shown in `server.js` ([file link](https://github.com/networked-aframe/networked-aframe/blob/master/server/index.js) in the repo)
2. Set up `networked-scene` attributes with `adapter: webrtc` and `audio: true`
3. Ran https local server and opened app with same roomId on two different tabs
4. The first tab was the only recognized "audio source" and was the only one transmitting audio
5. Second tab can hear the first one, but is not able to transmit audio

**TL:DR; CANNOT HEAR ECHO**

### Notes
1. The avatars appear
2. we can track each other's movement/rotation
3. only one person can be heard (the first person to join the room)

**Comment by silverslade** [comment]
H!
I confirm this issue.
Temporary switched to the 0.6.0 version with open easyrtc to get things work.

**Comment by haydenjameslee** [comment]
I'll have to look into this. Thanks for bringing it to my attention. PRs welcome.

**Comment by osresearch** [comment]
I'm encountering the same issue with patch #202 applied to keep all of the players visible.

**Comment by silverslade** [comment]
Hi !
I've done some further tests.
It occurs on the haydenlee.io page also: the first which enters in the room can listen and speak with microphone, the second which enters in the room can only listen ( ref: https://haydenlee.io/networked-aframe/basic-audio.html )
Tested on Firefox 76.0.1 and Chrome 81.0.4044.138 on Windows 10. 
Same issue: only the first user can listen and speak.
No problems with open easyrtc and networked-aframe 0.6.0 (the earlier version) with adapter: webrtc and audio: true

**Comment by djoksimo** [comment]
It seems as though `that.pendingAudioRequest[clientId]` never gets resolved when the second user joins:

https://github.com/networked-aframe/networked-aframe/blob/130687dc49859c8baf8de61176fbbb4249405ee9/src/adapters/naf-webrtc-adapter.js#L555
